disk-buffering/src/main/java/io/opentelemetry/contrib/disk/buffering/internal/serialization/deserializers/MetricDataDeserializer.java

```diff
@@ -28,4 +28,9 @@ public List<MetricData> deserialize(byte[] source) {
       throw new IllegalArgumentException(e);
     }
   }
+
+  @Override
+  public String signalType() {
+    return "metrics";
```

Comment:
Do you think we could reuse the same names already provided when building the exporters, like in the case of metrics [this one](https://github.com/open-telemetry/opentelemetry-java-contrib/blob/15e430779defecfedebfaa20f101f537d6a5849c/disk-buffering/src/main/java/io/opentelemetry/contrib/disk/buffering/MetricFromDiskExporter.java#L24), for example. Or if it's not feasible, then I think it'd be nice to at least have them as constants so that we don't have to repeat them.

Reply:
Sure, I went ahead and pulled them into an enum.